cli/cmd/root.go

```diff
@@ -64,7 +64,13 @@ This will prompt you for your Lacework account and a set of API access keys.`,
 				if cmd.HasParent() && cmd.Parent().Use == "configure" {
 					return nil
 				}
-				return cli.NewClient()
+
+				if err := cli.NewClient(); err != nil {
+					return err
+				}
+
+				// @afiune execute any necessary migration, if any
+				return cli.Migrations()
```

Comment:
Have you thought about moving cli.Migrations() call inside cli.NewClient()?

Reply:
No, mainly because the creation of an API client has nothing to do with the migration of the CLI configuration file.